docs/csharp/roslyn-sdk/tutorials/snippets/how-to-write-csharp-analyzer-code-fix/MakeConst/MakeConst/MakeConstAnalyzer.cs

```diff
@@ -118,7 +118,7 @@ private void AnalyzeNode(SyntaxNodeAnalysisContext context)
             }
 
             // <SnippetReportDiagnostic>
-            context.ReportDiagnostic(Diagnostic.Create(Rule, context.Node.GetLocation()));
+            context.ReportDiagnostic(Diagnostic.Create(Rule, context.Node.GetLocation(), localDeclaration.Declaration.Variables.First().Identifier.ValueText));
```

Comment:
@BillWagner @sharwell In case of multiple variables, e.g: `int x = 0, y = 0;`, the message will have the first name only. I assume it's acceptable, but wanted to explicitly note about that.